### ISSUE TITLE
Security issues

### ISSUE THREAD

**Issue by Youssef535** [TaxyAI/browser-extension#19]
`Taxy uses GPT-4 to control your browser and perform repetitive actions on your behalf` does it read and know all my information like my IP adress and the browser version... history ... we need to know

**Comment by arcticfly** [comment]
Hey @Youssef535, thanks for your attentiveness! Right now, we send a simplified version of the DOM to GPT-4, along with your instructions and a history of all actions it's performed for the given task so far. No IP address and browser version. Feel free to poke through the code in [performQuery.ts](https://github.com/TaxyAI/browser-extension/blob/main/src/helpers/performQuery.ts) to see what gets sent.

However, on the subject of security issues, there's a lot more we need to do to make this production-ready, such as guard-railing which sites Taxy can and can't interact with (for instance, you wouldn't want it messing around with your bank account) and setting a max-tokens limit for a given task. Would you mind creating a discussion on the subject? I think this is worth giving attention early on.

**Comment by Youssef535** [comment]
Yes sure thanks for your response @arcticfly  and i will discuss security issue with the devs.
